buildenv/jenkins/cleanupTestLib.groovy

```diff
@@ -8,41 +8,32 @@ def platformMap = [
     'zlinux' : '(ci.role.test||ci.role.test.fips)&&hw.arch.s390x&&sw.os.linux',
     'alinux' : '(ci.role.test||ci.role.test.fips)&&hw.arch.aarch64&&sw.os.linux',
     'win' : '(ci.role.test||ci.role.test.fips)&&hw.arch.x86&&sw.os.windows',
-    'aix' : '(ci.role.test||ci.role.test.fips)&&hw.arch.ppc64&&sw.os.aix'
+    'aix' : '(ci.role.test||ci.role.test.fips)&&hw.arch.ppc64&&sw.os.aix',
+    'zos':'ci.role.test&&hw.arch.s390x&&sw.os.zos'
 ]
 
 timeout(time: 72, unit: 'HOURS') {
     timestamps {
         def parallelJobs = [:]
-
         platformMap.each { platform, label ->
             echo "${platform}: ${label}"
-            if (platforms == 'all' || platforms == platform) {
                 if (params.PLATFORMS == 'all' || params.PLATFORMS == platform) {
-                    def cmd = "hostname; if [ -d /home/jenkins/${dependencyDir} ]; then ls -alt /home/jenkins/${dependencyDir}/; rm -rf /home/jenkins/${dependencyDir}/*; ls -alt /home/jenkins/${dependencyDir}/; else echo 'Directory does not exist: /home/jenkins/${dependencyDir}'; fi;"
-                    def cmd_clean_maven_libs = "if [ -d /home/jenkins/.m2 ]; then ls -alt /home/jenkins/.m2/; rm -rf /home/jenkins/.m2/*; ls -alt /home/jenkins/.m2/; else echo 'Directory does not exist: /home/jenkins/.m2/'; fi"
-                    if (platform == 'win') {
-                        cmd = "hostname; if [ -d C://Users//jenkins//${dependencyDir} ]; then ls -alt C://Users//jenkins//${dependencyDir}//;rm -rf C://Users//jenkins//${dependencyDir}//*;ls -alt C://Users//jenkins//${dependencyDir}//; else echo 'Directory does not exist: C://Users//jenkins//${dependencyDir}'; fi;"
-                        cmd_clean_maven_libs = "if [ -d C://Users//jenkins//.m2 ]; then ls -alt C://Users//jenkins//.m2/; rm -rf C://Users//jenkins//.m2/*; ls -alt C://Users//jenkins//.m2/; else echo 'Directory does not exist: C://Users//jenkins//.m2'; fi"
-                    } else if (platform == 'amac' || platform == 'xmac') {
-                        cmd = "hostname; if [ -d /Users/jenkins/${dependencyDir} ]; then ls -alt /Users/jenkins/${dependencyDir}/; rm -rf /Users/jenkins/${dependencyDir}/*; ls -alt /Users/jenkins/${dependencyDir}/; else echo 'Directory does not exist: /Users/jenkins/${dependencyDir}/'; fi;"
-                        cmd_clean_maven_libs = "if [ -d /Users/jenkins/.m2 ]; then ls -alt /Users/jenkins/.m2/; rm -rf /Users/jenkins/.m2/*; ls -alt /Users/jenkins/.m2/; else echo 'Directory does not exist: /Users/jenkins/.m2'; fi"
-                    }
-                    cmd += cmd_clean_maven_libs
-
                     parallelJobs[platform] = {
-                        stage("Build on ${platform}") {
-                            echo "Running job for platform: ${platform} with label: ${label}"
+                        node(label) {
+                            def userHome = (platform == 'win') ? env.USERPROFILE.replace('\\', '//') + '/' : env.HOME
```

Comment:
I am not sure `USERPROFILE` and `HOME` are standard [Jenkins Environment Variables](https://wiki.jenkins.io/display/JENKINS/Building+a+software+project#Buildingasoftwareproject-belowJenkinsSetEnvironmentVariables). Please use the standard variable `WORKSPACE` instead.